drivers/sdhc/sdhc_renesas_ra.c

```diff
@@ -0,0 +1,745 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_ra_sdhc
+
+#include <zephyr/kernel.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/irq.h>
+#include <soc.h>
+#include <zephyr/drivers/pinctrl.h>
```

Comment:
Duplicated include <zephyr/drivers/pinctrl.h>

Reply:
I removed it, thanks